nf_core/lint.py

```diff
@@ -5,15 +5,27 @@
 the nf-core community guidelines.
 """
 
+import click
+import datetime
 import logging
 import os
-import subprocess
 import re
 import requests
+import requests_cache
 import shlex
+import subprocess
+import tempfile
```

Comment:
Maybe we can sort the imports:

1.) legacy Python modules
2.) third party modules